app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/ImportFilesScreen.kt

```diff
@@ -104,11 +113,13 @@ private fun ImportFilesScreen(
                 TopAppBarButton.closeButton { closeActivity() },
             )
         },
-        topButton = { modifier ->
-            SendButton({ isImporting }, { importProgress }, modifier)
-        },
+        topButton = { modifier -> SendButton({ isImporting }, { importProgress }, modifier) },
         content = {
-            Column(Modifier.padding(horizontal = Margin.Medium, vertical = Margin.Large)) {
+            Column(
+                modifier = Modifier
+                    .padding(horizontal = Margin.Medium, vertical = Margin.Large)
+                    .verticalScroll(rememberScrollState()),
+            ) {
```

Comment:
What do you think about splitting the `content` of the column inside 3 or 4 methods: MyFiles(), TextFields(), SendBy() and AdvancedSettings() or something similar?

Reply:
That's a good idea but it should be done in another PR really dedicated to the importScreen.
This one is a component PR, I integrated it in the screen quickly to see the results and put it in better perspective, but it wasn't meant to replace the real PR for the whole screen